docs/UMLdiagrams/MenuLogicSequenceDiagram.puml

```diff
@@ -0,0 +1,54 @@
+@startuml
+!include style.puml
+
+Actor User as user USER_COLOR
+
+Participant ":SubLogic(MenuLogic)" as subLogic SUB_LOGIC_COLOR
```

Comment:
Should this just be ":MenuLogic" since `SubLogic` does not appear in our code, and `MenuLogic` is the entity that the user interacts with?